src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -56,6 +56,13 @@ abstract class AbstractObjectNormalizer extends AbstractNormalizer
      */
     public const DISABLE_TYPE_ENFORCEMENT = 'disable_type_enforcement';
 
+    /**
+     * While denormalizing, convert scalar types to the expected type.
+     *
+     * Enabled by default for the XML and CSV format, because all basic datatypes are represented as strings.
+     */
+    public const ENABLE_TYPE_CONVERSION = 'enable_type_conversion';
```

Comment:
Could you please update the `AbstractObjectNormalizerContextBuilder` and the related tests?